godot-bindings/src/lib.rs

```diff
@@ -161,3 +161,25 @@ pub fn emit_godot_version_cfg() {
         println!(r#"cargo:rustc-cfg=gdextension_exact_api="{major}.{minor}""#);
     }
 }
+
+// Function for safely removal of build directory. Workaround for errors happening during CI builds:
+// https://github.com/godot-rust/gdext/issues/616
+pub fn remove_dir_all_safe(path: &std::path::Path) {
```

Comment:
I would not use the term "safe" here; it's just unreliable or brittle without this.

Maybe `remove_dir_all_reliable` or something?